src/main/kotlin/app/revanced/patches/youtube/misc/integrations/fingerprints/EmbeddedPlayerFingerprint.kt

```diff
@@ -0,0 +1,13 @@
+package app.revanced.patches.youtube.misc.integrations.fingerprints
+
+import app.revanced.patcher.extensions.or
+import app.revanced.patches.shared.integrations.patch.AbstractIntegrationsPatch.IntegrationsFingerprint
+import org.jf.dexlib2.AccessFlags
+
+object EmbeddedPlayerFingerprint : IntegrationsFingerprint(
+    accessFlags = AccessFlags.PUBLIC or AccessFlags.STATIC,
+    returnType = "L",
+    parameters = listOf("L", "L", "Landroid/content/Context;"),
```

Comment:
Same as in other cases, only the first character is checked because the initial idea was to only check for the primitive types in Dalvik. This can be changed in ReVanced Patcher.